RecoTracker/TkTrackingRegions/python/GlobalTrackingRegionWithVertices_cfi.py

```diff
@@ -14,7 +14,19 @@
         useFoundVertices = cms.bool(True),
         useFakeVertices = cms.bool(False),
         maxNVertices = cms.int32(-1),
-        nSigmaZ = cms.double(4.0)
+        nSigmaZ = cms.double(4.0),
+
+        #parameters for HI collisions; doesn't do anything if all 3 booleans are False
+       
+        originRScaling4BigEvts = cms.bool(False),
```

Comment:
@makortel 
is this file still supposed to be used?
It's pretty confusing to have the auto-generated RecoTracker/TkTrackingRegions/globalTrackingRegionWithVertices_cfi.py and this file.

Reply:
@VinInn please note this "confusing" file as well.

Reply:
@slava77 This file is not used in pp tracking (and I was almost writing that it is obsolete), but I see it gets used for <= 2016 HI tracking (at least in sequences including `hiPixelPairStep`) in
https://github.com/cms-sw/cmssw/blob/3b4acba5c56c01824fcce1247605eebf8d61b486/RecoHI/HiTracking/python/hiPixelPairStep_cff.py#L50-L51
via
https://github.com/cms-sw/cmssw/blob/3b4acba5c56c01824fcce1247605eebf8d61b486/RecoTracker/TkSeedGenerator/python/GlobalSeedsFromPairsWithVertices_cff.py#L15-L24

Basically what should be done first is to migrate the phase0 `hiPixelPairStep` to the "new seeding framework".